### PR TITLE
Fix for unbounded KUBE_MANIFEST variables in upgarde.sh

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz @wojtek-t @gmarek

### CODE REVIEW

**Comment by wojtek-t** [comment]
@davidopp @kubernetes/goog-gke

**Comment by piosz** [comment]
LGTM

cc @davidopp @mikedanese @roberthbailey

File: cluster/common.sh [hunk 1]
```diff
@@ -328,9 +328,13 @@ function tars_from_version() {
   elif [[ ${KUBE_VERSION} =~ ${KUBE_RELEASE_VERSION_REGEX} ]]; then
     SERVER_BINARY_TAR_URL="https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/kubernetes-server-linux-amd64.tar.gz"
     SALT_TAR_URL="https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/kubernetes-salt.tar.gz"
+    KUBE_MANIFESTS_TAR_URL="${SERVER_BINARY_TAR_URL/server-linux-amd64/manifests}"
```
Comment (wojtek-t) [inline cluster/common.sh]: Can you add a TODO to clean this up?

**Comment by piosz** [comment]
ref #27764

File: cluster/common.sh (hunk 1 above)
Comment (mwielgus) [inline cluster/common.sh]: Done.

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**